Python/Algorithms/linked_list_structs.py

```diff
@@ -0,0 +1,185 @@
+
+
+class Node:
+    def __init__(self, val, next = None):
+        """
+        Creates a linked list node with given value `val` and points to `next` node.
+
+        params:
+            val (Any): Value of the node
+            next (Node): Next node in the linked list
```

Comment:
Where are the insert methods here

Reply:
I have implemented them in the latest Commit